docs/UsingNetlifyGuide.md

```diff
@@ -0,0 +1,30 @@
+## Using RepoSense with Netlify
+
+[Netlify](https://www.netlify.com) enables you to run RepoSense directly on your target repository. You can do everything on the browser itself and Netlify will handle running RepoSense for you. Therefore, as a user you don't need to download the latest jar file every time when wanting to run RepoSense.
+
+1. Fork the RepoSense repository using this [link](https://github.com/repoSense/RepoSense/fork).
+1. Set up Netlify in your forked repository. We recommend this [site](https://www.netlify.com/blog/2016/09/29/a-step-by-step-guide-deploying-on-netlify/) for ease of setup. Please refer to the picture below for **Step 5**.
+![Netlify setup](images/using-netlify-setup.png)
+1. Click on **Settings** in the top, choose **Build & deploy** from the left panel and scroll to **Build hooks**.
+![Build hooks](images/using-netlify-build-hooks.png)
+1. Click **Add build hook**, give your webhook a name, and choose the `master` branch to build. A Netlify URL will be generated.
+1. Go to your target repository (the repository you want to analyze) and click on **Settings**.
+1. Select **Webhooks** on left panel and click on **Add webhook**.
+![Add webhook](images/using-netlify-add-hook.png)
+1. Copy the Netlify URL and paste it in the URL form field.
+![Webhook url](images/using-netlify-url.png)
```

Comment:
If possible, for screenshots, provide it with an overview like the 3. for standardization and to ensure them that they are on the right track.

Also, although the build url is not really that secretive, do censor part of it to prevent any misuse and to highlight to the users that it should be kept safe.